### DISCUSSION TITLE
Discord invitation fails when reporting a bug

### DISCUSSION THREAD

**Discussion by rbochet** [nocodb/nocodb#7960] [General]
I've tried to report a bug from the interface but the Discord link doesnt work and I can't join the channel

![image](https://github.com/nocodb/nocodb/assets/56806/be891ce8-efff-4e97-9c4a-302c16677361)

**Comment by o1lab** [comment]
Thanks for binging this to our notice.

Please use [this](https://discord.gg/5RgZmkW) and ping us. Will update in the meanwhile.
